### ISSUE TITLE
Open Dialer, crash game

### ISSUE THREAD

**Issue by jordsta95** [McJtyMods/RFTools#400]
Dialer that hasn't dialed yet, upon opening crashes the game:
---- Minecraft Crash Report ----
// I feel sad now :(

Time: 04/12/15 12:58
Description: Ticking memory connection

java.lang.NullPointerException: Ticking memory connection
    at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:620)
    at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
    at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
    at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
    at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
    at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:620)
    at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
    at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
    at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)

-- Ticking connection --
Details:
    Connection: net.minecraft.network.NetworkManager@33203f6b
Stacktrace:
    at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
    at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Windows 10 (amd64) version 10.0
    Java Version: 1.8.0_65, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 662175728 bytes (631 MB) / 1631584256 bytes (1556 MB) up to 3817865216 bytes (3641 MB)
    JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx4096m
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 96
    FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1566 38 mods loaded, 38 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.05} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJAAAA   FML{7.10.99.99} [Forge Mod Loader](forge-1.7.10-10.13.4.1566-1.7.10-universal.jar) 
    UCHIJAAAA   Forge{10.13.4.1566} [Minecraft Forge](forge-1.7.10-10.13.4.1566-1.7.10-universal.jar) 
    UCHIJAAAA   CodeChickenCore{1.0.7.47} [CodeChicken Core](minecraft.jar) 
    UCHIJAAAA   NotEnoughItems{1.0.5.118} [Not Enough Items](NotEnoughItems-1.7.10-1.0.5.118-universal %281%29.jar) 
    UCHIJAAAA   voltzenginepreloader{0.0.1} [Voltz Engine Preloader](minecraft.jar) 
    UCHIJAAAA   BuildCraft|Core{7.1.13} [BuildCraft](buildcraft-7.1.13.jar) 
    UCHIJAAAA   BuildCraft|Builders{7.1.13} [BC Builders](buildcraft-7.1.13.jar) 
    UCHIJAAAA   BuildCraft|Transport{7.1.13} [BC Transport](buildcraft-7.1.13.jar) 
    UCHIJAAAA   BuildCraft|Energy{7.1.13} [BC Energy](buildcraft-7.1.13.jar) 
    UCHIJAAAA   BuildCraft|Silicon{7.1.13} [BC Silicon](buildcraft-7.1.13.jar) 
    UCHIJAAAA   BuildCraft|Robotics{7.1.13} [BC Robotics](buildcraft-7.1.13.jar) 
    UCHIJAAAA   BuildCraft|Factory{7.1.13} [BC Factory](buildcraft-7.1.13.jar) 
    UCHIJAAAA   factorization.notify{1.0} [Factorization Notification System](Factorization-1.7.10-0.8.99d.jar) 
    UCHIJAAAA   factorization.dimensionalSlice{0.8.99d} [Factorization Dimensional Slices](Factorization-1.7.10-0.8.99d.jar) 
    UCHIJAAAA   factorization{0.8.99d} [Factorization](Factorization-1.7.10-0.8.99d.jar) 
    UCHIJAAAA   factorization.misc{0.8.99d} [Factorization Miscellaneous Nonsense](Factorization-1.7.10-0.8.99d.jar) 
    UCHIJAAAA   fz.scrap{0.8.99d} [Scrap](Factorization-1.7.10-0.8.99d.jar) 
    UCHIJAAAA   factorization.truth{0.8.99d} [Truth](Factorization-1.7.10-0.8.99d.jar) 
    UCHIJAAAA   fastleafdecay{1.3} [Fast Leaf Decay](FastLeafDecay-1.7.10-1.3 %281%29.jar) 
    UCHIJAAAA   Trajectory{0.1.2} [Trajectory](Trajectory-1.7.10-0.1.2-universal.jar) 
    UCHIJAAAA   framez{0.1.1} [Framez 2](Framez2-1.7.10-0.1.1-hotfix-universal.jar) 
    UCHIJAAAA   framez_compat{1.0} [Framez Compat](Framez2-1.7.10-0.1.1-hotfix-universal.jar) 
    UCHIJAAAA   VoltzEngine{0.6.15.177} [Voltz Engine](VoltzEngine-1.7.10-0.6.15b177-dev-universal.jar) 
    UCHIJAAAA   icbm{2.6.0.81} [ICBM](ICBM-1.7.10-2.6.0b81-dev-universal.jar) 
    UCHIJAAAA   IGWMod{1.1.6-23} [In-Game wiki Mod](IGW-Mod-1.7.10-1.1.6-23-universal.jar) 
    UCHIJAAAA   Railcraft{9.8.0.0} [Railcraft](Railcraft_1.7.10-9.8.0.0 %281%29.jar) 
    UCHIJAAAA   ImmersiveEngineering{0.6.2} [Immersive Engineering](ImmersiveEngineering-0.6.2 %281%29.jar) 
    UCHIJAAAA   inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks](InventoryTweaks-1.59-dev-152 %281%29.jar) 
    UCHIJAAAA   journeymap{5.1.1} [JourneyMap](journeymap-1.7.10-5.1.1-unlimited.jar) 
    UCHIJAAAA   LogisticsPipes{0.9.3.69} [Logistics Pipes](logisticspipes-0.9.3.69.jar) 
    UCHIJAAAA   PneumaticCraft{1.12.1-144} [PneumaticCraft](PneumaticCraft-1.7.10-1.12.1-144-universal.jar) 
    UCHIJAAAA   Waila{1.5.10} [Waila](Waila-1.5.10_1.7.10 %281%29.jar) 
    UCHIJAAAA   RefinedRelocation{1.1.12} [Refined Relocation](RefinedRelocation-mc1.7.10-1.1.12.jar) 
    UCHIJAAAA   rftools{4.13} [RFTools](rftools-4.13.jar) 
    UCHIJAAAA   StevesWorkshop{0.5.1} [Steve's Workshop](StevesWorkshop-0.5.1.jar) 
    UCHIJAAAA   TrajectoryCompat{0.1} [Trajectory Compat](Trajectory-1.7.10-0.1.2-universal.jar) 
    UCHIJAAAA   TwilightForest{2.3.7} [The Twilight Forest](twilightforest-1.7.10-2.3.7.jar) 
    GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
    Profiler Position: N/A (disabled)
    Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    Player Count: 1 / 8; [EntityPlayerMP['jordsta95'/2651, l='New World', x=331.64, y=66.00, z=265.70]]
    Type: Integrated Server (map_client.txt)
    Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by McJty** [comment]
This is a bug that occurs from time to time but not only with RFTools. I've seen it happen with other mods too. No clue what causes it.

**Comment by guruzaheer** [comment]
I'm having a similar, if not the same crash, when I right click a dialer with a tinker's construct tool in my hand.

**Comment by Esirza** [comment]
I just ran into this crash about thirty minutes ago. My dialing device was working fine until I decided to set it up in combination with buttons on a screen. After that point I haven't been able to access the GUI. As well, I'm not getting a full client crash, just a world/server crash. I'm guessing on one of the attempts a crash report finally generated and it mentioned "Exception ticking world

java.lang.IllegalArgumentException: Could not get provider type for dimension 6, does not exist"
I tried using the probe to debug the issue but that crashes the world as well. Currently, I'm getting ready to back up my world and see what happens if the dimension is deleted altogether.
